Confus/Common/Networking/Connection.cpp

```diff
@@ -81,5 +91,19 @@ namespace ConfusServer
 			a_InputStream.Read(contents);
 			std::cout << "Message received: " << contents << std::endl;
 		}
+
+        void Connection::sendPacketToAllClients(RakNet::BitStream& a_BitStream)
+        {
+            auto connectionCount = getConnectionCount();
```

Comment:
Perhaps you can refactor the existing code to retrieve all the connections using one function, rather than duplicating it